Pipfile

```diff
@@ -16,3 +16,4 @@ synapseclient = "~=2.7"
 pandas = "<1.5"
 moto = "~=4.1"
 datacompy = "~=0.8"
+docker = "~=6.1"
```

Comment:
Why do we need this?

Reply:
It's dependency of using `mock_lambda` from `moto3` but for some reason installing `moto3` doesn't automatically install `docker` as a dependency